src/content/effective-dart/documentation.md

```diff
@@ -548,14 +588,15 @@ think.
 
 When documenting a member for a class, you often need to refer back to the
 object the member is being called on. Using "the" can be ambiguous.
+Prefer having some qualifier after "this", a sole "this" can ambiguous too.
```

Comment:
(The "good" example of "The value this wraps." doesn't read well. I find that having an unqualifiied "this", "that", or a plain "it" usually makes documentation harder to read.)

Reply:
I agree. The qualifier helps quite a bit. Nice!